src/jobs/veracode_pipeline_scan.yml

```diff
@@ -57,3 +57,7 @@ steps:
       event: fail
       channel: << parameters.slack_channel >>
       custom: <<include(files/slack_failed_with_results.tpl.json)>>
```

Comment:
results.json is cached and restored in this job - so this solution looks to be doing the kind of the same thing.